repo/packages/N/namerd/0/config.json

```diff
@@ -1,66 +0,0 @@
-{
```

Comment:
Generally, we like to keep packages around indefinitely (so that folks running the older versions can still use them). Unfortunately, we don't have a way to mark `deprecated` right now. What do you think about leaving this around and adding some text to the description and pre-install notes to update users to it being deprecated? I just opened a bug to track getting something like this added to the product (https://dcosjira.atlassian.net/browse/DCOS-402)